### PR TITLE
Fix entry point conflict with actually exist file

### LINKED ISSUE #547: Failed to import graphql - library seems to be completely broken
## Failing module

- **GitHub**: 
- **npm**: graphql

```js
import { GraphQLError } from 'https://esm.sh/graphql@16.6.0/graphql'
```

## Error message

After running `deno run` I got this:

```
error: Uncaught SyntaxError: The requested module '/v111/graphql@16.6.0/deno/graphql.js' does not provide an export named 'GraphQLError'
```

## Additional info

- **esm.sh version**: No idea.
- **Deno version**:
deno 1.31.2 (release, aarch64-apple-darwin)
v8 11.0.226.19
typescript 4.9.4

Module used to work until about 8th of March 2023. After that Deno actually complained about hash change and after reloading everything error started to appear.

### PR DESCRIPTION
**Pull request by Justinidlerz**
Fix #547

### CODE REVIEW

File: server/build.go [hunk 1]
```diff
@@ -42,7 +42,17 @@ func (task *BuildTask) ID() string {
 
 	if pkg.Submodule != "" {
 		name = pkg.Submodule
+	} else {
+		pack, _ := fetchPackageInfo(pkg.Name, pkg.Version)
```
Comment (ije) [inline server/build.go:46]: - pls handle the error in case `pack` is nil.
- consider the "." of `exports`

**Review by ije — COMMENTED** [review]
and package css path is broken, check the ci details

File: server/build.go (hunk 1 above)
Comment (Justinidlerz) [reply server/build.go:46]: Sure,  Will fix these cases tomorrow
Fixing `pack` is nil is difficult in the `ID` method coz the method is called in many places. 
I think can refactor the logic of the build later.

**Comment by Justinidlerz** [comment]
> and package CSS path is broken, check the ci details

Yup, Do you want to follow up the CSS file path with the entry path?

**Pull request closed without merging**